sql/repl_failsafe.cc

```diff
@@ -243,6 +249,10 @@ bool show_slave_hosts(THD* thd)
                         Item_empty_string(thd, "Gtid_State_Ack", GTID_MAX_STR_LENGTH),
                         thd->mem_root);
 
+ field_list.push_back(new (mem_root)
+                        Item_empty_string(thd, "Sync", GTID_MAX_STR_LENGTH),
```

Comment:
`Sync` alone doesn't provide a purpose; I think `Sync Status` would be more meaningful.